modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenSecurity.java

```diff
@@ -28,7 +28,7 @@ public class CodegenSecurity {
     public String description;
     public String type;
     public String scheme;
-    public Boolean isBasic, isOAuth, isApiKey, isOpenId;
+    public Boolean isBasic, isOAuth, isApiKey, isOpenIdConnect;
```

Comment:
Sorry I may not be very clear before.

I think using `OpenID` is fine to represent the security definition type.

Reply:
isn't `OpenId` different from `OpenIdConnect` ? @wing328

Reply:
also according to the spec, it's `openIdConnect`

Reply:
it's the same thing.

> also according to the spec, it's openIdConnect

Yup, so `isOAuth` should be named `isOAuth2` instead if we simply follow that (but it's not the case)

I want to avoid mustache tags with long name if possible. That's why I suggest `isOpenID` instead.

Reply:
that's why I simply want to keep the naming introduced in https://github.com/OpenAPITools/openapi-generator/pull/15407

it looks straightforward to me.

what do you guys think?

I don't think using `OpenIdConnect` at all if that's what you guys prefer.

Reply:
I mean I don't mind, but it might confuse people to have some places with OpenId and others with OpenIdConnect, considering they are 2 different specs.
but an argument can be made here is that OpenId is deprecated, and it defaults to OpenIdConnect anyway.

so both is fine by me

Reply:
So I corrected all my rebase errors and left isOpenID too ease mustache templates. Just tell me If I should use isOpenIdConnect instead.